src/custom_api_response.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+class CustomApiResponse:
+    """
+    this class represents response data structure for returning response
+    from functions.
+    """
+
+    def __init__(self, data):
+        for key in data:
+            self.key = data[key]
```

Comment:
BTW, I am pretty this does not what you expect it to do:
```Python
from custom_api_response import CustomApiResponse
input = {
  "data_url": "https://archive.softwareheritage.org/api/1/content/sha1_git:fe95a46679d128ff167b7c55df5d02356c5a1ae1/raw/",
  "filetype_url": "https://archive.softwareheritage.org/api/1/content/sha1_git:fe95a46679d128ff167b7c55df5d02356c5a1ae1/filetype/",
  "language_url": "https://archive.softwareheritage.org/api/1/content/sha1_git:fe95a46679d128ff167b7c55df5d02356c5a1ae1/language/",
  "length": 151810,
  "license_url": "https://archive.softwareheritage.org/api/1/content/sha1_git:fe95a46679d128ff167b7c55df5d02356c5a1ae1/license/",
  "status": "visible",
}

car = CustomApiResponse(input)

car.data_url
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'CustomApiResponse' object has no attribute 'data_url'

car.key
'visible'
```

so this is not right at all

Reply:
Yeah got that, my bad.
Thanks.